cypress/integration/specialFeatures/scriptSwitching/testsForCanonicalOnly.js

```diff
@@ -0,0 +1,75 @@
+import visitPage from '../../../support/helpers/visitPage';
+import {
+  allVariantAssertions,
+  nonCookieVariantAssertions,
+} from '../utilities/scriptSwitchingJourneyAssertions';
+import {
+  clickScriptSwitcher,
+  clickHomePageLink,
+  clickPromoLinkOnHomePage,
+} from '../utilities/scriptSwitchingJourneyActions';
+import {
+  getPrivacyBannerAccept,
+  getCookieBannerAccept,
+} from '../utilities/cookiePrivacyBanner';
+
+export default ({
+  serviceId,
+  serviceName,
+  pageType,
+  path,
+  variant,
+  otherVariant,
+}) => {
+  describe(`Script Switching - ${serviceId} - ${pageType} - ${path}`, () => {
+    beforeEach(() => {
+      cy.clearCookies();
+      visitPage(path, pageType);
+    });
+
+    it(`should change to the correct script when switching script between ${variant} and ${otherVariant}`, () => {
+      // Accept privacy banner
+      getPrivacyBannerAccept(serviceId, variant).click();
+
+      // Accept cookie banner
+      getCookieBannerAccept(serviceId, variant).click();
+
+      cy.log(
+        `Asserting script switch button, url and document lang against variant: ${variant}`,
+      );
+      nonCookieVariantAssertions(serviceName, variant);
```

Comment:
Should this be `allVariantAssertions` instead of `nonCookieVariantAssertions`

Reply:
See this comment https://github.com/bbc/simorgh/pull/6051#discussion_r417237129

Reply:
Cool, happy to leave as is.